### ISSUE TITLE
HTML file uploaded as a blog post gets incorrect title

### ISSUE THREAD

**Issue by sillasgonzaga** [daattali/beautiful-jekyll#57]
I've written a blog post that uses two htmlwidget packages, DT and visNetwork. Therefore, a md file would not be a good option for me.

However, I can't get the blog to fix the title. I generated the file using knitr and Rstudio and changed the YAML parameters as follows:

---
title: "Meta-análise R (2): Analisando Pacotes R como indivíduos em uma rede social"
author: "Sillas Teixeira Gonzaga"
date: "May 30, 2016"
layout: post
comments: true
output: html_document
---

However, the string title is now what appears what the home page of the blog, but the string after the date in the file name. So if I name the html file as 2016-05-30-post3.html, then post3 will be the header of that post on the blog homepage.

When I noted this, I copied the title from the YAML header to the file name, but it didn't work properly as it lowercased it.

[Link to the blog page](http://sillasgonzaga.github.io/)
[Link to the file in the repo](https://github.com/sillasgonzaga/sillasgonzaga.github.io/blob/master/_posts/2016-05-30-Meta-an%C3%A1lise%20R%20%282%29:%20Analisando%20Pacotes%20R%20como%20indiv%C3%ADduos%20em%20uma%20rede%20social.html)

So, how do I fix the title of a blog entry created by an html file?

**Comment by daattali** [comment]
I don't understand the problem.  The title of the page that I'm seeing is "Meta Análise r (2): analisando pacotes r como indivíduos em uma rede social" -- is that not correct?

**Comment by sillasgonzaga** [comment]
If you compare the title of the page with the name of the file that generated it, you will see that they are different.

Page title: Meta Análise r (2): analisando pacotes r como indivíduos em uma rede social
File name: 2016-05-30-Meta-análise R (2): Analisando Pacotes R como indivíduos em uma rede social.html

You see, some words got lowercased. Posting html files do not seem to be as straightforward as md ones.

**Comment by daattali** [comment]
I see now.  I tried Googling that problem and it looks like it's a Jekyll bug.  (Or maybe not a bug, maybe it's on purpose).  Anyway, they say that the only way to fix that is to include a title in the YAML of the html file. One of the links I found discussion this: https://github.com/jekyll/jekyll/issues/313

I'm closing this issue since it's a general Jekyll issue and not something related to this project
